src/helpers/helpers.core.js

```diff
@@ -306,6 +306,20 @@ var helpers = {
 
 		ChartElement.__super__ = me.prototype;
 		return ChartElement;
+	},
+
+	/**
+	 * If argument is string, return it. If it is an array, join its elements using
+	 * a space string (' ').
+	 * @private
+	 */
+	_joinIfArray: function(stringOrArray) {
+		if (!stringOrArray) {
```

Comment:
I think this will convert an empty string to `null`. I'm not sure we want that?

Reply:
You're right, fixed to return parameter object in all cases where it's not an array.